src/awkward/contents/unionarray.py

```diff
@@ -1064,14 +1064,8 @@ def _reverse_merge(self, other):
                 AssertionError("FIXME: handle UnionArray with more than 127 contents")
             )
 
-        parameters = ak.forms.form._parameters_union(
-            self._parameters,
-            other._parameters,
-            exclude=ak.forms.form.reserved_nominal_parameters,
-        )
-
         return ak.contents.UnionArray.simplified(
-            tags, index, contents, parameters=parameters
+            tags, index, contents, parameters=self._parameters
```

Comment:
What if `self` and `other` are _both_ UnionArrays? Should their parameters be merged, then?

Reply:
Simplified handles this case, which is nice!